%
%
% UCSD Doctoral Dissertation Template
% -----------------------------------
% http://ucsd-thesis.googlecode.com
%
%


%% REQUIRED FIELDS -- Replace with the values appropriate to you

% No symbols, formulas, superscripts, or Greek letters are allowed
% in your title.
\title{Methods for Integrative Analysis of RNA Binding Proteins}

\author{Gabriel Asbury Pratt}
\degreeyear{2018}

% Master's Degree theses will NOT be formatted properly with this file.
\degreetitle{Doctor of Philosophy}

\field{Bioinformatics and Systems Biology}
%\specialization{Anthropogeny}  % If you have a specialization, add it here

\chair{Professor Gene Yeo}
% Uncomment the next line iff you have a Co-Chair
\cochair{Professor Sheng Zhong}
%
% Or, uncomment the next line iff you have two equal Co-Chairs.
%\cochairs{Professor Chair Masterish}{Professor Chair Masterish}

%  The rest of the committee members  must be alphabetized by last name.
\othermembers{
Professor Jens Lykke-Andersen\\
Professor Bing Ren\\
Professor Joseph Simpson\\
}
\numberofmembers{5} % |chair| + |cochair| + |othermembers|


%% START THE FRONTMATTER
%
\begin{frontmatter}

%% TITLE PAGES
%
%  This command generates the title, copyright, and signature pages.
%
\makefrontmatter

%% DEDICATION
%
%  You have three choices here:
%    1. Use the ``dedication'' environment.
%       Put in the text you want, and everything will be formated for
%       you. You'll get a perfectly respectable dedication page.
%
%
%    2. Use the ``mydedication'' environment.  If you don't like the
%       formatting of option 1, use this environment and format things
%       however you wish.
%
%    3. If you don't want a dedication, it's not required.
%
%
\begin{dedication}
  To my parents who taught me to be curious, my mentors who showed me the cool things to be curious about, my friends who kept me balanced through out all of this.  
\end{dedication}


% \begin{mydedication} % You are responsible for formatting here.
%   \vspace{1in}
%   \begin{flushleft}
% 	To me.
%   \end{flushleft}
%
%   \vspace{2in}
%   \begin{center}
% 	And you.
%   \end{center}
%
%   \vspace{2in}
%   \begin{flushright}
% 	Which equals us.
%   \end{flushright}
% \end{mydedication}



%% EPIGRAPH
%
%  The same choices that applied to the dedication apply here.
%
\begin{epigraph} % The style file will position the text for you.
  \emph{Progress isn't made by early risers.
                 It's made by lazy men trying to find easier ways to do something}\\
  ---Robert Heinlein
\end{epigraph}

% \begin{myepigraph} % You position the text yourself.
%   \vfil
%   \begin{center}
%     {\bf Think! It ain't illegal yet.}
%
% 	\emph{---George Clinton}
%   \end{center}
% \end{myepigraph}


%% SETUP THE TABLE OF CONTENTS
%
\tableofcontents
\listoffigures  % Comment if you don't have any figures
%\listoftables   % Comment if you don't have any tables



%% ACKNOWLEDGEMENTS
%
%  While technically optional, you probably have someone to thank.
%  Also, a paragraph acknowledging all coauthors and publishers (if
%  you have any) is required in the acknowledgements page and as the
%  last paragraph of text at the end of each respective chapter. See
%  the OGS Formatting Manual for more information.
%
\begin{acknowledgements}
To all my co-authors and lab mates I    couldn't have done this without you.    Specifically I want to call out Mike Lovci, who taught me how to analyze CLIP-seq data.  I’ve leaned on your tools more than you know.  Katannya Kapeli who was the first person to drag me through a full biological story.  Finally I wouldn’t have finished my PhD with out Eric Van Nostrand, who taught me everything else, but most importantly taught me how to walk carefully through analyses instead of just running ahead. I wouldn’t be half the researcher I am without out.

I also want acknowledge the people who put me on this path. My high school Genetics teacher Penny Pagels, who first introduced me to the field.  Chuck Murry who took a chance on a first year undergrad who wanted to work in a lab. My two main undergraduate mentors, Jonathan Golob, who exposed me to everything bioinformatics could be and Sharron Paige, who let me help on the coolest projects.

A short acknowledgement, cannot properly recognize everyone who has helped me along the way, or encompass what I’ve learned from everyone.  I am eternally grateful for all the time and energy you all have invested, and hope to pay it back some day.

\end{acknowledgements}


%% VITA
%
%  A brief vita is required in a doctoral thesis. See the OGS
%  Formatting Manual for more information.
%
\begin{vitapage}
\begin{vita}
  \item[2011] B.~S. in Computer Science, University of Washington, Seattle
  \item[2018] Ph.~D. in Bioinformatics and Systems Biology, University of California, San Diego
\end{vita}
\begin{publications}

        \item Gabriel A. Pratt, Eric L. Van Nostrand, Brian A. Yee, Alain Domissy, Steven M. Blue, Chelsea Gelboin-Burkhart, Thai B. Nguyen, Ines Rabano, Ruth Wang, Balaji Sundararaman, Keri Garcia, Rebecca Stanton, Gene W. Yeo. ``Guidelines and Best Practices for enhanced CLIP experiments and analysis''. \emph{In Submission}, 2017
        \item Eric L Van Nostrand§, Peter Freese§, Gabriel A Pratt§, Xiaofeng Wang§, Xintao Wei§, Rui Xiao§, Steven M Blue, Daniel Dominguez, Neal A.L. Cody, Sara Olson, Balaji Sundararaman, Lijun Zhan, Cassandra Bazile, Louis Philip Benoit Bouvrette, Jiayu Chen, Michael O Duff, Keri E. Garcia, Chelsea Gelboin-Burkhart, Abigail Hochman, Nicole J Lambert, Hairi Li, Thai B Nguyen, Tsultrim Palden, Ines Rabano, Shashank Sathe, Rebecca Stanton, Julie Bergalet, Bing Zhou, Amanda Su, Ruth Wang, Brian A. Yee, Ashley L Louie, Stefan Aigner, Xiang-dong Fu, Eric Lecuyer, Christopher B. Burge, Brenton R. Graveley, Gene W. Yeo. ``A Large-Scale Binding and Functional Map of Human RNA Binding Proteins'' \emph{In Submission}, 2017
        \item Eric L Van Nostrand, Chelsea Gelboin-Burkhart, Ruth Wang, Gabriel A Pratt, Steven M Blue, Gene W Yeo. ``CRISPR/Cas9-mediated integration enables TAG-eCLIP of endogenously tagged RNA binding proteins''. \emph{Methods}, 2016
      \item Fernando J Martinez, Gabriel A Pratt, Eric L Van Nostrand, Ranjan Batra, Stephanie C Huelga, Katannya Kapeli, Peter Freese, Seung J Chun, Karen Ling, Chelsea Gelboin-Burkhart, Layla Fijany, Harrison C Wang, Julia K Nussbacher, Sara M Broski, Hong Joo Kim, Rea Lardelli, Balaji Sundararaman, John P Donohue, Ashkan Javaherian, Jens Lykke-Andersen, Steven Finkbeiner, C Frank Bennett, Manuel Ares, Christopher B Burge, J Paul Taylor, Frank Rigo, Gene W Yeo. ``Protein-RNA Networks Regulated by Normal and ALS-Associated Mutant HNRNPA2B1 in the Nervous System.'' \emph{Neuron}, 2016
      \item Kristopher W Brannan, Wenhao Jin, Stephanie C Huelga, Charles AS Banks, Joshua M Gilmore, Laurence Florens, Michael P Washburn, Eric L Van Nostrand, Gabriel A Pratt, Marie K Schwinn, Danette L Daniels, Gene W Yeo. ``SONAR Discovers RNA-Binding Proteins from Analysis of Large-Scale Protein-Protein Interactomes.'' \emph{Molecular Cell}, 2016
      \item Katannya Kapeli§, Gabriel A. Pratt§, Anthony Q. Vu, Kasey R. Hutt, Fernando J. Martinez, Balaji Sundararaman, Ranjan Batra, Peter Freese, Nicole J. Lambert, Stephanie C. Huelga, Seung Chun, Tiffany Y. Liang, Jeremy Chang, John P. Donohue, Lily Shiue, Jiayu Zhang, Haining Zhu, Franca Cambi, Edward Kasarskis, Manuel Ares Jr., Christopher B. Burge, John Ravits, Frank Rigo, Gene W. Yeo. ``Distinct and shared molecular targets and functions of ALS-associated TDP-43, FUS, and TAF15 revealed by comprehensive multi-system integrative analyses.'' \emph{Nature Communications}, 2016
      \item Stefan Rentas, Nicholas T Holzapfel, Muluken S Belew, Gabriel A Pratt, Veronique Voisin, Brian T Wilhelm, Gary D Bader, Gene W Yeo, Kristin J Hope. ``Musashi-2 attenuates AHR signalling to expand human haematopoietic stem cells.'' \emph{Nature}, 2016
      \item Anne E Conway, Eric L Van Nostrand, Gabriel A Pratt, Stefan Aigner, Melissa L Wilbert, Balaji Sundararaman, Peter Freese, Nicole J Lambert, Shashank Sathe, Tiffany Y Liang, Anthony Essex, Severine Landais, Christopher B Burge, D Leanne Jones, Gene W Yeo. ``Enhanced CLIP Uncovers IMP Protein-RNA Targets in Human Pluripotent Stem Cells Important for Cell Adhesion and Survival.'' \emph{Cell Reports}, 2016
      \item Eric L Van Nostrand, Gabriel A Pratt, Alexander A Shishkin, Chelsea Gelboin-Burkhart, Mark Y Fang, Balaji Sundararaman, Steven M Blue, Thai B Nguyen, Christine Surka, Keri Elkins, Rebecca Stanton, Frank Rigo, Mitchell Guttman, Gene W Yeo. ``Robust transcriptome-wide discovery of RNA-binding protein binding sites with enhanced CLIP (eCLIP).'' \emph{Nature Methods}, 2016
      \item Balaji Sundararaman, Lijun Zhan, Steven M Blue, Rebecca Stanton, Keri Elkins, Sara Olson, Xintao Wei, Eric L Van Nostrand, Gabriel A Pratt, Stephanie C Huelga, Brendan M Smalec, Xiaofeng Wang, Eurie L Hong, Jean M Davidson, Eric Lécuyer, Brenton R Graveley, Gene W Yeo. ``Resources for the comprehensive discovery of functional RNA elements.'' \emph{Molecular Cell}, 2016
      \item T. Hung, G. A. Pratt, B. Sundararaman, M. J. Townsend, C. Chaivorapol, T. Bhangale, R. R. Graham, W. Ortmann, L. A. Criswell, G. W. Yeo, T. W. Behrens. ``The Ro60 autoantigen binds endogenous retroelements and regulates inflammatory gene expression.'' Science, 2015
      \item Suzanne R. Lee, Gabriel Pratt, Fernando Martinez, Gene W. Yeo, Jens Lykke-Andersen. ``Target discrimination in nonsense-mediated mRNA decay requires the ATPase activity of Upf1.'' \emph{Molecular Cell}, 2015
      \item Singh G, Gabriel Pratt, Yeo GW, Moore MJ. ``The Clothes Make the mRNA: Past and Present Trends in mRNP Fashion.'' \emph{Annual Review of Biochemistry}, 2015
      \item Lovci MT, Ghanem D, Marr H, Arnold J, Gee S, Parra M, Liang TY, Stark TJ, Gehman LT, Hoon S, Massirer KB, Gabriel Pratt, Black DL, Gray JW, Conboy JG, Yeo GW. ``Rbfox proteins regulate alternative mRNA splicing through evolutionarily conserved RNA bridges.'' \emph{Nature Structural and Molecular Biology}, 2013
      \item Sharon L. Paige, Sean Thomas, Cristi Stoick-Cooper, Hao Wang, Richard Sandstrom4, Lisa Maves, Lil Pabon, Hans Reinecke, Gabriel Pratt, Gordon Keller, Randall T. Moon, John Stamatoyannopoulos, and Charles E. Murry. ``A Temporal Chromatin Signature in Human Embryonic Stem Cells Identifies Novel Regulators of Cardiovascular Development.'' \emph{Cell} 2013
      \item Golob, J. L., Kumar, R. M., Guenther, M. G., Pabon, L. M., Pratt, G. A., Loring, J. F., Laurent, L. C., Young, R. A., and Murry, C. E. ``Evidence That Gene Activation and Silencing during Stem Cell Differentiation Requires a Transcriptionally Paused Intermediate State.'' \emph{PLoS ONE}, 2011

\end{publications}
\end{vitapage}


%% ABSTRACT
%
%  Doctoral dissertation abstracts should not exceed 350 words.
%   The abstract may continue to a second page if necessary.
%
\begin{abstract}
Cross-linking immunoprecipitation (CLIP) has been used to profile the binding sites of over 100 RNA binding proteins (RBPs).  However computational pipelines, quality control metrics, and downstream analyses needed to process CLIP data at scale have yet to be well defined. Here we describe in detail the characterization of a single RBP, TAF15, which is known to be involved in amyotrophic lateral sclerosis.  We detail computational processing techniques, including integration of RNA-seq, microarray splicing, RNA bind-n-seq (RBNS) and stability assays to understand the function TAF15 in mouse and human brains.  Next we describe how to scale analyses from one RBP to many.  We present our ENCODE eCLIP processing pipeline, enabling users to go from raw reads to significant, reproducible peaks, that can be directly compared against ENCODE eCLIP experiments. In particular, we discuss processing steps designed to address common artifacts, including quantifying unique RNA fragments bound by both unique genomic- and repetitive element-mapped reads. Using manual quality annotation of 350 ENCODE eCLIP experiments, we develop metrics for quality assessment of eCLIP experiments before and after sequencing, including recommendations for library yield, number of unique fragments in library, binding information, and biological reproducibility. In particular, we quantify the linkage between sequencing depth and peak discovery, and derive methods for estimating sequencing depth based on pre-sequencing metrics. Finally we provide recommendations for the integration of RBP binding and RNA-seq experiments to generate splicing maps. These pipelines and QC metrics enable large-scale processing and analysis of eCLIP data, and enable rigorous and standard analysis of RBP binding data.  Finally we describe results from analysis of additional RBPs that illustrate the utility of studying the dynamics of RBP binding in different contexts.  Specifically we detail how understanding the location of UPF1 binding lead to a better understanding of the mechanism of action for UPF1 in nonsense medicated decay. We also detail how information on Musahi 2 binding improved understanding of the mechanism of haematopoietic stem cells expansion.
\end{abstract}


\end{frontmatter}
